### PR TITLE
Bring up-to-date with internal code

### PR DESCRIPTION
**Pull request by apolyudov**
Signed-off-by: Alexey Polyudov <apolyudov@google.com>
Change-Id: I86c64d081912f6a03c0bd86d706e8ccedf350978

### CODE REVIEW

File: cpp/src/securemessage/crypto_ops_openssl.cc [hunk 1]
```diff
@@ -19,6 +19,8 @@
 #include <openssl/ec.h>
```
Comment (leshi) [inline cpp/src/securemessage/crypto_ops_openssl.cc:19]: This file merged a bit messy.  Can you confirm that this branch builds and runs all the tests (and that they pass)?

File: script/oss.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python3
```
[the remark this answers is not in the document]
Comment (apolyudov) [reply script/oss.py:1]: done

File: script/handle_oss.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (apolyudov) [reply script/handle_oss.sh:1]: done

File: .gitmodules [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[submodule "third-party/protobuf"]
```
[the remark this answers is not in the document]
Comment (apolyudov) [reply .gitmodules:1]: done

File: cpp/src/securemessage/crypto_ops_openssl.cc (hunk 1 above)
Comment (apolyudov) [reply cpp/src/securemessage/crypto_ops_openssl.cc:19]: this big-ish diff is the result of internal review. I verified it then (at the time of writing) and verified it again now. Unit tests are passing.

**Comment by leshi** [comment]
LGTM

**Pull request merged**